spec/requests/v1/recipes_spec.rb

```diff
@@ -3,11 +3,47 @@
 require 'swagger_helper'
 
 RSpec.describe 'V1::Recipes', type: :request do
+  let(:user_credentials) { create(:user).create_new_auth_token }
+  let(:HTTP_ACCESS_TOKEN) { user_credentials['access-token'] }
+  let(:HTTP_TOKEN_TYPE) { user_credentials['token-type'] }
+  let(:HTTP_CLIENT) { user_credentials['client'] }
+  let(:HTTP_EXPIRY) { user_credentials['expiry'] }
+  let(:HTTP_UID) { user_credentials['uid'] }
+
   path '/v1/recipes' do
     post 'Creates a recipe' do
       tags 'Recipes'
       description 'Creates an instance of Recipe.'
       consumes 'application/json'
+      parameter name: :HTTP_ACCESS_TOKEN,
+                in: :header,
+                type: :string,
+                example: 'abcd1dMVlvW2BT67xIAS_A',
+                required: true
+      parameter name: :HTTP_TOKEN_TYPE,
+                in: :header,
+                type: :string,
+                example: 'Bearer',
+                required: true
+
+      parameter name: :HTTP_CLIENT,
+                in: :header,
+                type: :string,
+                example: 'LSJEVZ7Pq6DX5LXvOWMq1w',
+                required: true
+
+      parameter name: :HTTP_EXPIRY,
+                in: :header,
+                type: :string,
+                example: '1519086891',
+                required: true
+
+      parameter name: :HTTP_UID,
+                in: :header,
+                type: :string,
+                example: 'user@mail.com',
+                required: true
+
       parameter name: :recipe, in: :body, schema: {
         type: :object,
         properties: {
```

Comment:
I dont see a title here, isn't that mandatory?

Reply:
Of fudge, we never had the validation for it to have a title, right?

Reply:
We do, but in this case I only need an ID I make the call